node/src/test_utils/mod.rs

```diff
@@ -577,63 +579,92 @@ pub mod unshared_test_utils {
         pub assertions: Box<dyn FnOnce(&mut A) + Send>,
     }
 
-    pub fn make_default_persistent_configuration() -> PersistentConfigurationMock {
-        PersistentConfigurationMock::new()
-            .past_neighbors_result(Ok(None))
-            .gas_price_result(Ok(1))
-            .mapping_protocol_result(Ok(None))
-    }
-
     pub fn make_simplified_multi_config<'a, const T: usize>(args: [&str; T]) -> MultiConfig<'a> {
         let mut app_args = vec!["MASQNode".to_string()];
         app_args.append(&mut array_of_borrows_to_vec(&args));
         let arg_matches = app_node().get_matches_from_safe(app_args).unwrap();
         MultiConfig::new_test_only(arg_matches)
     }
 
-    pub const ZERO: u32 = 0b0;
-    pub const MAPPING_PROTOCOL: u32 = 0b000010;
-    pub const ACCOUNTANT_CONFIG_PARAMS: u32 = 0b000100;
-    pub const RATE_PACK: u32 = 0b001000;
+    pub enum PCField {
```

Comment:
I appreciate the translation to this more friendly design. Well done.

Reply:
Thanks!